packages/appsync-modelgen-plugin/src/__tests__/visitors/appsync-swift-visitor.test.ts

```diff
@@ -2131,5 +2146,35 @@ describe('AppSyncSwiftVisitor', () => {
         }"
       `);
     });
+    it('should keep the original init method if timestamp is enabled but no read-only fields found', () => {
```

Comment:
nit: maybe change the description of this test case to something like _"should keep the original init method if timestamp ff is enabled and schema has explicit timestamp fields"_